transifex/requirements.txt

```diff
@@ -1,5 +1,11 @@
-Django==1.11.2
-edx-i18n-tools==0.3.9
+Babel==1.3
+Django==1.10.7
+django-babel-underscore==0.5.2
+edx-i18n-tools==0.4.1
```

Comment:
I'm assuming that this will also change re: https://github.com/edx/i18n-tools/pull/63/files